ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/TransactionServiceVisibilityIT.scala

```diff
@@ -97,6 +97,13 @@ class TransactionServiceVisibilityIT extends LedgerTestSuite {
           def go(eventId: String): Unit = {
             eventsToObserve.remove(eventId) match {
               case Some(TreeEvent(Exercised(exercisedEvent))) =>
+                val expected = exercisedEvent.childEventIds.sortBy(_.split(':')(1).toInt)
+                val actual = exercisedEvent.childEventIds
+                assertEquals(
```

Comment:
This test case is suitable to test this condition: IouTrade is being exercised (which boil down to transaction with exercise node with many children)

Reply:
This test will no longer be run for older versions of platform in the cross-version check. Although the probability of it being a problem are slim. I'd much rather we had a dedicated new test for the sort.

Reply:
AFAIK this needs to be only excluded for the transient versions -- no exclusion eventually

Reply:
Right, thanks for the clarification: moved it to a separate test